content/docs/goFSH/running/_index.md

```diff
@@ -18,18 +18,28 @@ where options include the following (in any order):
 -l, --log-level <level>           specify the level of log messages: error, warn, info (default), debug
 -d, --dependency <dependency...>  specify dependencies to be loaded using format dependencyId@version (FHIR R4 included by default)
 -v, --version                     print goFSH version
+-s, --style                       specify how the output is organized into files: group-by-fsh-type (default), group-by-profile, single-file, file-per-definition
+-f, --fshing-trip                 run SUSHI on the output of GoFSH and generate a comparison of the round trip results
+-i, --installed-sushi             use the locally installed version of SUSHI when generating comparisons with the "-f" option
 -h, --help                        output usage information
 ```
 
-While GoFSH is running, it will print status messages as it processes your project files.
+While GoFSH is running, it will print status messages as it processes your project files. The following sections give further detail on using certain options.
+
+### `style`
+The `style` option has four values:
+* `group-by-fsh-type`: Definitions are written to files based on what type of FSH definition they are (Alias, Profile, Extension, etc.). This is the default choice.
+* `group-by-profile`:  Profiles are each written to an individual file. Instances and Invariants that pertain only to a certain Profile are then included in the same file as that Profile. The remaining definitions are grouped as in the `group-by-fsh-type` option.
+* `single-file`: All definitions are written to one file.
+* `file-per-definition`: Each standalone FSH definition is written to an individual file. Only Aliases are combined into one `aliases.fsh` file.
+
+### `fshing-trip`
+If this flag is added, after GoFSH runs, SUSHI will run on the output of GoFSH. The output of SUSHI will then be compared to the original input to GoFSH (FHIR is compared to FHIR), and a visualization of differences between the original input and the SUSHI output will be created in `<output-folder>/fshing-trip-comparison.html`. If the `--installed-sushi` flag is set, then this process will use whichever version of SUSHI you have globally installed. Otherwise GoFSH will use SUSHI version 1.1.0.
```

Comment:
I don't think we should reference the exact SUSHI version here, else we'll have to update this with nearly every GoFSH release.  Perhaps:
> Otherwise GoFSH will use its own built-in version of SUSHI (which may not be the latest version available).